includes/shortcodes/class-wc-shortcode-products.php

```diff
@@ -534,10 +543,10 @@ protected function product_loop() {
 
 		ob_start();
 
-		if ( $products_ids ) {
+		if ( $products_ids && $products_ids['ids'] ) {
```

Comment:
Since we are changing what `get_products_ids` returns, do we need a new method?

Reply:
I don't believe so. It's protected and it's only been around since 3.2.4. Before that it was called `get_products`. The only place it's used in in `product_loop` below.

Reply:
I prefer we change the name of the method again.